manifests/init.pp

```diff
@@ -71,6 +71,8 @@
 #
 # $ca_key_password::              CA key password
 #
+# $ciphers::                      Allowed ciphers for ssl connection. Array or strings
```

Comment:
Should this be Array o**f** strings? In that case could also add a validate_array? Then you don't have to cast it in the template.